applications/CompressiblePotentialFlowApplication/compressible_potential_flow_application.h

```diff
@@ -100,6 +100,7 @@ class KRATOS_API(COMPRESSIBLE_POTENTIAL_FLOW_APPLICATION) KratosCompressiblePote
     ///@{
 
     const IncompressiblePotentialFlowElement<2,3> mIncompressiblePotentialFlowElement2D3N;
+	const IncompressiblePotentialFlowElement<3, 4> mIncompressiblePotentialFlowElement3D4N;
```

Comment:
the indentation format here is different from the rest

Reply:
done!